.travis.yml

```diff
@@ -30,10 +30,10 @@ matrix:
         - rustup target add x86_64-fortanix-unknown-sgx x86_64-unknown-linux-musl
       script:
         - cargo test --verbose --all --exclude sgxs-loaders
-        - cargo test --verbose -p sgx-isa --features sgxstd -Z package-features --target x86_64-fortanix-unknown-sgx --no-run
-        - cargo test --verbose -p sgxs-tools --features pe2sgxs --bin isgx-pe2sgx -Z package-features
-        - cargo test --verbose -p dcap-ql --features link -Z package-features
-        - cargo test --verbose -p dcap-ql --features verify -Z package-features
+        - cargo test --verbose -p sgx-isa --features sgxstd --target x86_64-fortanix-unknown-sgx --no-run
```

Comment:
Shouldn't you use the new `--features package/feature` syntax?

Reply:
I don't think so, judging by the build success.

Reply:
Ah sorry, misread your comment. Yes I can do that